testing/kfctl/scripts/delete_existing_cluster.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Can this be used here.
https://github.com/kubeflow/testing/blob/a841aa0b2b1f6b64e06f5ab2be9f6df2ba3b214e/py/kubeflow/testing/util.py#L196-L207

Reply:
This is more straightforward to achieve.
I will try to use it here.